### PR TITLE
feat: add zh-hans for interacting with editor part

### PR DESCRIPTION
**Pull request by Ele-Peng**
The PR aiming to provide the zh-hans doc guide.
It's including four parts:
	1. interacting with editor
	2. commands
	3. styling
	4. keyboard shortcuts

looking forward to getting feedback from you.

### CODE REVIEW

File: gh-pages/pages/guide/commands/index.zh-hans.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+// call command
+commandManager.call(WrapInHeading); // turn to h1 by default
+commandManager.call(WrapInHeading, 2); // turn to h2
+```
+
+---
+
+## 内部命令
+
+### 基准
```
Comment (Saul-Mirone) [inline gh-pages/pages/guide/commands/index.zh-hans.md]: Commonmark就不要翻译了

File: gh-pages/pages/guide/interacting-with-editor/index.zh-hans.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+};
+
+new Editor()
+    .config((ctx) => {
+        ctx.set(listenerCtx, listener);
+    })
+    .use(listener);
+```
+
+接着，我们可以使用获取到的 `jsonOutput` 作为浏览器的默认值配置：
```
Comment (Saul-Mirone) [inline gh-pages/pages/guide/interacting-with-editor/index.zh-hans.md]: 浏览器应该为编辑器

File: gh-pages/pages/guide/keyboard-shortcuts/index.zh-hans.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# 快捷键
```
Comment (Saul-Mirone) [inline gh-pages/pages/guide/keyboard-shortcuts/index.zh-hans.md:1]: Please format this file with prettier

**Review by Saul-Mirone — COMMENTED** [review]
Please format all these files using prettier

**Comment by Ele-Peng** [comment]
> Please format all these files using prettier

has resolved.

**Review by Saul-Mirone — APPROVED** [review]
OK

**Pull request merged**